tests/baselines/reference/signatureHelpCommentsCommentParsing.baseline

```diff
@@ -425,13 +425,13 @@
 // | This is function comment
 // |     And aligned with 4 space char margin
 // | @param a this is info about a
-// | spanning on two lines and aligned perfectly
+// |                   spanning on two lines and aligned perfectly
 // | @param b this is info about b
-// | spanning on two lines and aligned perfectly
-// | spanning one more line alined perfectly
-// |     spanning another line with more margin
+// |                   spanning on two lines and aligned perfectly
+// |                   spanning one more line alined perfectly
+// |                       spanning another line with more margin
 // | @param c this is info about b
-// | not aligned text about parameter will eat only one space
+// |  not aligned text about parameter will eat only one space
```

Comment:
This to me looks very unexpected; I would expect that common leading spaces are chopped off, such that this does not change. This contrasts with the other jsdoc server tests where this seems to work.